### PR TITLE
add vision bucket to internvl-3-14b

### PR DESCRIPTION
**Pull request by libinta**
## Essential Elements of an Effective PR Description Checklist
- [ ] The purpose of the PR, such as "Fix some issue (link existing issues this PR will resolve)".
- [ ] The test plan, such as providing test command.
- [ ] The test results, such as pasting the results comparison before and after, or e2e results

### CODE REVIEW

File: vllm/worker/hpu_model_runner.py [hunk 1]
```diff
@@ -2709,17 +2733,28 @@ def create_dummy_multi_modal_seq_group_metadata(self, group_id, img_args,
...
+                multi_modal_data = {
+                    "pixel_values_flat":
+                    pixel_values.to(torch.bfloat16),
+                    "image_num_patches":
+                    torch.tensor([pixel_values.shape[0]], dtype=torch.int32),
+                    "image_token_id":
+                    torch.tensor([self.image_token_id], dtype=torch.int64),
+                }
+            else:
+                logger.warning("No support for other models yet")
```
Comment (adobrzyn) [inline vllm/worker/hpu_model_runner.py:2752]: Couple of lines earlier we have this assert:
`assert self.model_is_mrope or self.is_mm_optimized, \
            ("Warmup compatible with Qwen2vl/Gemma3 models")`
1. If we are supporting InternVLChatModel than we should add it there
2. Curently we would say that we only support warmup for qwen and gemma and for quen we would end up with "No support"
3. Shouldn't that be assert? Or maybe we should skip warmup as whole if we have other model? Currently we only log warning - easy to miss and ignore

**Review by michalkuligowski — APPROVED** [review]

**Comment by piotrbocian** [comment]
Fork will not be released in 1.24 so this PR to main will not be released anywhere.
Please cherry-pick to proper 1.23.0_next branch.
